src/code/z_play.c

```diff
@@ -18,12 +18,16 @@ void Play_SpawnScene(PlayState* this, s32 sceneId, s32 spawn);
 
 // This macro prints the number "1" with a file and line number if R_ENABLE_PLAY_LOGS is enabled.
 // For example, it can be used to trace the play state execution at a high level.
+#if OOT_DEBUG
 #define PLAY_LOG(line)                            \
     do {                                          \
         if (R_ENABLE_PLAY_LOGS) {                 \
             LOG_NUM("1", 1, "../z_play.c", line); \
         }                                         \
     } while (0)
+#else
+#define PLAY_LOG(line) (void)0
+#endif
```

Comment:
does the debug check in the if work?

Reply:
Not at first glance, not even as `&& OOT_DEBUG`. Breaks matching in a few place, both retail and debug